sdk/typescript/src/cryptography/keypair.ts

```diff
@@ -85,9 +85,21 @@ export abstract class BaseSigner {
 	abstract getPublicKey(): PublicKey;
 }
 
```

Comment:
importing bcs from `@mysten/bcs` instead of using the bcs from the SDK should resolve the cyclic import lint issue in this file